install.sh

```diff
@@ -943,11 +943,12 @@ ohai "Downloading and installing Homebrew..."
   cd "${HOMEBREW_REPOSITORY}" >/dev/null || return
 
   # we do it in four steps to avoid merge errors when reinstalling
-  execute "${USABLE_GIT}" "-c" "init.defaultBranch=master" "init" "--quiet"
+  execute "${USABLE_GIT}" "init" "--quiet"
```

Comment:
I believe the reason we've had this here is to override user settings when people have it set to `main` or something else in their Git config. (Git's default is `master`.) So I think we should continue to override this but instead change the branch name to `main`?